apps/frontend/src/pages/projects/components/project-card/project-card.tsx

```diff
@@ -2,23 +2,33 @@ import { NavLink } from "react-router-dom";
 
 import { AppRoute } from "~/libs/enums/enums.js";
 import { configureString } from "~/libs/helpers/helpers.js";
+import { useCallback } from "~/libs/hooks/hooks.js";
 import { type ProjectGetAllItemResponseDto } from "~/modules/projects/projects.js";
 
+import { ProjectMenu } from "../components.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	onEditClick: (project: ProjectGetAllItemResponseDto) => void;
 	project: ProjectGetAllItemResponseDto;
 };
 
-const ProjectCard = ({ project }: Properties): JSX.Element => {
+const ProjectCard = ({ onEditClick, project }: Properties): JSX.Element => {
 	const projectRoute = configureString(AppRoute.PROJECT, {
 		id: project.id.toString(),
 	});
 
+	const handleEditClick = useCallback(() => {
+		onEditClick(project);
+	}, [onEditClick, project]);
+
 	return (
-		<NavLink className={styles["project"] ?? ""} to={projectRoute}>
-			{project.name}
-		</NavLink>
+		<div className={styles["project"]}>
+			<NavLink className={styles["project-link"] ?? ""} to={projectRoute}>
```

Comment:
We agreed on [this approach](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/167#discussion_r1743086886)
```suggestion
			<NavLink className={styles["project-link"] as string} to={projectRoute}>
```